oci-provision/README.md

```diff
@@ -10,7 +10,7 @@ It is quite convenient when you want to quickly create instances in a reproducib
 This small project illustrates usage of the OCI CLI to create instances from the _Platform_ catalog or the _Marketplace_.
 
 # Prerequisites
-1. A computer running Linux or OS-X.
+1. A computer running Linux or macOS.
```

Comment:
No love for Windows Subsystem for Linux? :)

Reply:
No Windows machine to test ;)

Reply:
Tested!